geo-location.html

```diff
@@ -38,6 +44,16 @@
 
     properties: {
       /**
+       * Indicates whether `Geolocation API` is supported or not.
+       */
+      apiSupported: {
```

Comment:
what about just `supported`?

Reply:
@ebidel IMO, `api-supported` is more clear than just `supported`. Many web components use `api-supported`, e.g.:

https://webcomponents.org/element/jorgecasar/network-information/elements/network-information#property-apiSupported

https://webcomponents.org/element/abdonrd/device-orientation/elements/device-orientation#property-apiSupported

Reply:
fair enough